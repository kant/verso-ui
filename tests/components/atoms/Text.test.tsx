import { jsx } from '@emotion/react'
import renderer from 'react-test-renderer'
import { Text } from 'src/components'

test('Snapshot Test: Text', () => {
  const component = renderer.create(
    <Text element="a" fontWeight="light" fontSize="medium">
      I am a string
    </Text>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('Snapshot Test: Text', () => {
  const component = renderer.create(
    <Text element="p" fontWeight="medium" fontSize="large">
      I am a string
    </Text>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
