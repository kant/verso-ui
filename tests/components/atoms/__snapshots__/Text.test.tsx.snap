// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Text 1`] = `
<a
  className="css-zkrtb7-Text"
>
  I am a string
</a>
`;

exports[`Snapshot Test: Text 2`] = `
<p
  className="css-1aktc3o-Text"
>
  I am a string
</p>
`;
